### PR TITLE
Extend EDID capabilities with CAE info blocks (#142)

### PR DESCRIPTION
**Pull request by 404ag**
This is a cherry picked from R2 branch

### CODE REVIEW

**Comment by mhughesacn** [comment]
Hi Artur: As this is on master, please use Metrological in the header of the new file, not RDK Management.  Thanks!
- Martin Hughes, RDK CMF Compliance Team

**Comment by artur-gebicz-ml** [comment]
> Hi Artur: As this is on master, please use Metrological in the header of the new file, not RDK Management. Thanks!
> 
> * Martin Hughes, RDK CMF Compliance Team
My mistake. Fixed.

File: Source/displayinfo/include/ExtendedDisplayIdentification.h [hunk 1]
```diff
@@ -303,23 +734,139 @@ namespace Plugin {
...
+                }
+            }
+            return colorFormatMap;
+        }
+
+        uint8_t SupportedColorFormats() const {
+            uint8_t colorFormatMap = 0;
+
+            Iterator segment = Iterator(_segments);
+            while(segment.Next() == true) {
```
Comment (pwielders) [inline Source/displayinfo/include/ExtendedDisplayIdentification.h:813]: I see a repetition of 6 times the same code to find an CEA::extension_tag. Might be good to move this to method, before every one starts to deploy this copy-Paste OO :-)

**Review by pwielders — CHANGES_REQUESTED** [review]

File: Source/displayinfo/include/ExtendedDisplayIdentification.h (hunk 1 above)
Comment (404ag) [reply Source/displayinfo/include/ExtendedDisplayIdentification.h]: Done :)

**Review by pwielders — APPROVED** [review]

**Pull request merged**